script.js

```diff
@@ -1,14 +1,349 @@
+    sign-ups required
+    personalized reads; no accounts or 
+    sign-ups required. Has been the
+     industry's standard.`,
+    technologies: ['HTML', 'Boostrap', 'Ruby'],
+    button: 'See Project',
+  },
+  {
+    title: 'Profesional Art Printing Data',
+    image: 'images/Img Placeholder (2).png',
+    description: `A daily selection of privately personalized reads;
+    no accounts or sign-ups required.
+    has been the industry standard.`,
+    technologies: ['HTML', 'Boostrap', 'Ruby'],
+    button: 'See Project',
+  },
+  {
+    title: 'Data Dashboard Healthcare',
+    image: 'images/Img Placeholder (4).png',
+    description: `A daily selection of privately personalized reads;
+   no accounts or sign-ups required.
+    has been the industry standard.`,
+    technologies: ['HTML', 'Boostrap', 'Ruby'],
+    button: 'See Project',
+  },
+  {
+    title: 'Website Portfolio',
+    image: 'images/Img Placeholder (5).png',
+    description: `A daily selection of privately 
+    personalized reads; no accounts or 
+    sign-ups required
+    personalized reads; no accounts or 
+    sign-ups required. Has been the
+     industry's standard.`,
+    technologies: ['HTML', 'Boostrap', 'Ruby'],
+    button: 'See Project',
+  },
+];
+
+const portfolio = [
+  {
+    title: 'My Recent Works',
+    header: 'Multi Post Stories',
+    technologies: ['CSS', 'HTML', 'Boostrap', 'Ruby'],
+    image: 'images/Img Placeholder.png',
+    imageMob: 'images/Img Placeholder (3).png',
+    description: `A daily selection
+    of privately personalized reads; 
+    no accounts or sign-ups <br> required. 
+    This has been the industry's 
+    standard dummy <br> text ever since 
+    the 1500s, when an unknown
+     printer took a <br>
+    standard dummy text.`,
+    button: 'See Project',
+  },
+];
+
+const popup = [
+  {
+    title: 'Keeping track of hundreds of components website',
+    titleMob: 'Multi Post Stories',
+    button: ['HTML', 'Boostrap', 'Ruby on rails'],
+    image: 'images/aya.png',
+    imageMob: 'images/snapsmall.png',
+    description: `Lorem Ipsum is simply dummy text of
+     the printing and typesetting industry. Lorem Ipsum has been the 
+     industry's standard dummy text ever since the 1500s, when an unknown
+     make a type specimen book.
+       It has survived not only five centuries,
+     but also the leap into electronic typesetting, remaining essent`,
+    descriptionMob: `Lorem Ipsum is simply dummy text of
+    the printing and typesetting industry. Lorem Ipsum has been the 
+    industry's standard dummy text ever since the 1500s, when an unknown
+      make a type specimen book.
+      It has survived not only five centuries,
+    but also the leap into electronic typesetting, remaining essent`,
+    links: ['See live', 'See source'],
+    linkImg: ['images/link.png', 'images/Vector.png'],
+    linkSource: ['https://blessman-newton.github.io/Portfolio/', 'https://github.com/Blessman-Newton/Portfolio'],
+
+  },
+];
```

Comment:
- [ ] According to the project requirement, you are expected to dynamically display the project and make it such that when a particular "See project" button is clicked, the project data from the project array is used to populate the popup display card. In simple terms, Modal popup data should be populated with the data from the projects array. _My observation of your project at the moment is that when I click on the "Data Dashboard Healthcare" or "Website Portfolio" project card, I still see the data you have in this popup array._ 
 
 You can check an example of students' projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95 and https://github.com/nicupop729/Portfolio/pull/11/commits/aabed37cc2208f52d0bd081a4cae8cdc83c49636. as a guide.